.travis.yml

```diff
@@ -1,47 +1,42 @@
 language: java
 sudo: required
-services: [ docker ]
-
-jdk: [ oraclejdk8 ]
-
+services:
+- docker
+jdk:
+- oraclejdk8
 env:
   global:
   - PATH="${PATH}:${HOME}/.local/bin"
-
 install:
 - pip install --user awscli
-
 script:
 - set -o pipefail
 - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
     gem install travis-cron_tools;
     ruby .travis-cron.rb;
   fi;
-- if [ ! -z "$AWS_ACCESS_KEY" ]; then
-    aws cloudformation validate-template --template-body "file:////${PWD}/deploy/aws-ecs/cloudformation.json" | jq . ;
-  fi;
-
+- if [ ! -z "$AWS_ACCESS_KEY" ]; then aws cloudformation validate-template --template-body
+  "file:////${PWD}/deploy/aws-ecs/cloudformation.json" | jq . ; fi;
 after_success:
 - export GROUP=weaveworksdemos
-- ./push.sh
-
+- "./push.sh"
 deploy:
   provider: s3
-  access_key_id: $AWS_ACCESS_KEY_ID
-  secret_access_key: $AWS_SECRET_ACCESS_KEY
+  access_key_id: "$AWS_ACCESS_KEY_ID"
```

Comment:
it was working before. But I think this is the recommended way by Travis, because I reprocessed the config using the travis command line, and it made a few style changes.